### PR TITLE
#65 add field status label and cover image url

### PR DESCRIPTION
**Pull request by setiadijoe**
- add field status label
- add field cover image url
- add env variabel storage_api public

### CODE REVIEW

File: endpoint/response.go [hunk 1]
```diff
@@ -75,20 +83,29 @@ func EncodePhonebook(data []*model.Phonebook) []*Phonebook {
...
 
 		result = append(result, encodeData)
 	}
 
 	return result
 }
+
+// GetStatusLabel ...
+var GetStatusLabel = map[int64]map[string]string{
+	-1: {"en": "status deleted", "ina": "Dihapus"},
```
Comment (rindibudiaramdhan) [inline endpoint/response.go:108]:
```suggestion
	-1: {"en": "Status Deleted", "ina": "Dihapus"},
```

I suggest to make it capital.

**Review by rindibudiaramdhan — CHANGES_REQUESTED** [review]

File: endpoint/response.go (hunk 1 above)
Comment (setiadijoe) [reply endpoint/response.go:108]: image
image
I think, on the exists contract, it use sentence case

**Review by firmanJS — APPROVED** [review]

**Comment by firmanJS** [comment]
LGTM !

**Review by rindibudiaramdhan — APPROVED** [review]

**Comment by setiadijoe** [comment]
> LGTM !

apa ini mang?

**Comment by firmanJS** [comment]
> > LGTM !
> 
> apa ini mang?

LGTM = let's go to merge 
speak english dude @setiadijoe wkwkwk :rofl:

**Comment by setiadijoe** [comment]
> > > LGTM !
> > 
> > 
> > apa ini mang?
> 
> LGTM = let's go to merge
> speak english dude @setiadijoe wkwkwk 🤣

I will use bilingual with Sunda

**Comment by rindibudiaramdhan** [comment]
> > > LGTM !
> > 
> > 
> > apa ini mang?
> 
> LGTM = let's go to merge
> speak english dude @setiadijoe wkwkwk 🤣

wkwkwk is not english TBH 🤣🤣🤣🤣

File: endpoint/response.go [hunk 2]
```diff
@@ -75,20 +83,29 @@ func EncodePhonebook(data []*model.Phonebook) []*Phonebook {
...
+	0:  {"en": "Not Active", "ina": "Tidak Aktif"},
+	10: {"en": "Active", "ina": "Aktif"},
```
Comment (aldidoanta) [inline endpoint/response.go:110]: @setiadijoe Kudos for the localization :+1:

Should we use [ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) to denote language? For example, language code for Indonesian/Bahasa Indonesia is `id`.

**Comment by aldidoanta** [comment]
> LGTM = let's go to merge

@firmanJS I thought LGTM is "Looks Good to Me" :laughing: . Perhaps there are different interpretations for LGTM.

File: endpoint/response.go [hunk 3]
```diff
@@ -75,20 +83,29 @@ func EncodePhonebook(data []*model.Phonebook) []*Phonebook {
...
+			ID:            v.ID,
+			PhoneNumbers:  phoneNumbers,
+			Description:   v.Description,
+			Name:          v.Name,
+			Address:       v.Address,
+			CoverImageURL: fmt.Sprintf("%s/%s", cfg.AppStoragePublicURL, v.CoverImageURL),
+			Latitude:      v.Latitude,
+			Longitude:     v.Longitude,
+			Status:        v.Status,
+			StatusLabel:   GetStatusLabel[v.Status]["ina"],
```
Comment (aldidoanta) [inline endpoint/response.go:95]: @setiadijoe Some leftovers :grin: 
```suggestion
			StatusLabel:   GetStatusLabel[v.Status]["id"],
```

File: endpoint/phonebook.go [hunk 4]
```diff
@@ -76,8 +77,9 @@ func MakeGetDetail(ctx context.Context, usecase usecase.Provider) endpoint.Endpo
 			Village:        resp.Village,
 			Latitude:       resp.Latitude,
 			Longitude:      resp.Longitude,
-			CoverImagePath: resp.CoverImagePath,
+			CoverImagePath: fmt.Sprintf("%s/%s", cfg.AppStoragePublicURL, resp.CoverImagePath),
 			Status:         resp.Status,
+			StatusLabel:    GetStatusLabel[resp.Status]["ina"],
```
Comment (aldidoanta) [inline endpoint/phonebook.go:82]: @setiadijoe Some leftovers :grin: 
```suggestion
			StatusLabel:   GetStatusLabel[v.Status]["id"],
```

Comment (setiadijoe) [reply endpoint/phonebook.go]: i had created on new commit

**Pull request merged**